docs/markdown/connecting-to-testnet.md

```diff
@@ -0,0 +1,61 @@
+# Connecting to an ILP Testnet
+
+This tutorial will help you start moneyd locally on your computer and connect it to an ILP router on the testnet.
+
+This tutorial describes how to:
+
+1. Configure and start [`moneyd`](https://github.com/interledgerjs/moneyd) to connect to an 
+[ILP router](https://github.com/interledgerjs/ilp-connector) on the testnet
+2. Send value to a [payment pointer](https://interledger.org/rfcs/0026-payment-pointers/) on the testnet
+
+## Before you begin
+For this tutorial, you will need `moneyd` and SPSP installed on your computer. You can learn how to install `moneyd` 
+and SPSP from the [Getting Started](getting-started.md) tutorial.
+
+NOTE: For this tutorial you do not need to use real cryptocurrency. Since you will be sending value through an 
+ILP router on the testnet, funds will be used from a [testnet faucet](https://xrpl.org/xrp-test-net-faucet.html) and have no real-world value.
+
+## Configuring moneyd
+NOTE: Before configuring `moneyd`, make sure to delete the `.moneyd.test.json` config file if it already exists on 
+your computer.
+
+To configure moneyd: 
+
+1. Open a terminal and use the following command:
+```shell
+moneyd xrp:configure --testnet --advanced
+```
+
+2. When prompted for the `BTP host of parent connector`, enter:
+```shell
+js.xpring.dev
+```
+
+3. Press enter for other default options.
```

Comment:
Consider: "Press enter for _all_ other default options."

Reply:
https://github.com/interledger/interledger.github.io/pull/108/commits/259f9fdeeda358cb53f45bb23c88eab35d44af39#diff-95b449acc2e692458b91372a5b6e33e7R34